base/applications/rapps/settingsdlg.cpp

```diff
@@ -9,6 +9,20 @@
 
 SETTINGS_INFO NewSettingsInfo;
 
+static int CALLBACK
+BrowseFolderCallback(HWND hwnd, UINT uMsg, LPARAM lParam, LPARAM lpData)
+{
+    switch (uMsg)
+    {
+        case BFFM_INITIALIZED:
+            SendMessageW(hwnd, BFFM_SETSELECTIONW, TRUE, lpData);
+            break;
+        case BFFM_VALIDATEFAILED:
+            return TRUE;
```

Comment:
In that case, does the browse dialog show an error popup?

Reply:
No but it does prevent Ok from closing the dialog (ROS does not yet implement validation so there this has no effect).